docs/csharp/language-reference/builtin-types/unmanaged-types.md

```diff
@@ -1,20 +1,28 @@
 ---
 title: "Unmanaged types - C# reference"
-ms.date: 07/23/2019
+ms.date: 09/06/2019
 helpviewer_keywords: 
   - "unmanaged type [C#]"
 ---
 # Unmanaged types (C# reference)
 
-An **unmanaged type** is any type that isn't a reference type or constructed type (a type that includes at least one type argument), and doesn't contain reference type or constructed type fields at any level of nesting. In other words, an unmanaged type is one of the following:
+A type is an **unmanaged type** if it's any of the following types:
 
 - `sbyte`, `byte`, `short`, `ushort`, `int`, `uint`, `long`, `ulong`, `char`, `float`, `double`, `decimal`, or `bool`
 - Any [enum](../keywords/enum.md) type
 - Any [pointer](../../programming-guide/unsafe-code-pointers/pointer-types.md) type
-- Any user-defined [struct](../keywords/struct.md) type that is not a constructed type and contains fields of unmanaged types only
+- Any user-defined [struct](../keywords/struct.md) type that contains fields of unmanaged types only and, in C# 7.3 and earlier, is not a constructed type (a type that includes at least one type argument)
 
 Beginning with C# 7.3, you can use the [`unmanaged` constraint](../../programming-guide/generics/constraints-on-type-parameters.md#unmanaged-constraint) to specify that a type parameter is a non-pointer unmanaged type.
 
+Beginning with C# 8.0, a *constructed* struct type that contains fields of unmanaged types only is also unmanaged, as the following example shows:
+
+[!code-csharp[unmanaged constructed types](~/samples/csharp/language-reference/builtin-types/UnmanagedTypes.cs#ProgramExample)]
+
+A generic struct may be the source of both unmanaged and not unmanaged constructed types. The preceding example defines a generic struct `Coords<T>` and presents the examples of unmanaged constructed types. The example of not an unmanaged type is `Coords<object>`. It's not unmanaged because it has the fields of the `object` type, which is not unmanaged. If you want *any* constructed type to be an unmanaged type, use the `unmanaged` constraint in the definition of a generic struct:
```

Comment:
nit:

```suggestion
A generic struct may be the source of both unmanaged and not unmanaged constructed types. The preceding example defines a generic struct `Coords<T>` and presents the examples of unmanaged constructed types. The example of not an unmanaged type is `Coords<object>`. It's not unmanaged because it has the fields of the `object` type, which is not unmanaged. If you want *all* constructed types to be unmanaged types, use the `unmanaged` constraint in the definition of a generic struct:
```

Reply:
Thank you @BillWagner